exchanges/orderbook/orderbook_test.go

```diff
@@ -507,108 +455,104 @@ func TestSorting(t *testing.T) {
 }
 
 func deploySliceOrdered() Levels {
-	ts := make([]Level, 1000)
+	lvls := make([]Level, 1000)
 	for i := range 1000 {
-		ts[i] = Level{Amount: 1, Price: float64(i + 1), ID: rand.Int63()} //nolint:gosec // Not needed in tests
+		lvls[i] = Level{Amount: 1, Price: float64(i + 1), ID: rand.Int63()} //nolint:gosec // Not needed in tests
 	}
-	return ts
+	return lvls
 }
 
 func TestReverse(t *testing.T) {
-	s := Book{
-		VerifyOrderbook: true,
-	}
-
-	s.Bids = deploySliceOrdered()
-	require.Len(t, s.Bids, 1000)
-	assert.ErrorIs(t, s.Verify(), errPriceOutOfOrder)
+	b := Book{VerifyOrderbook: true, Bids: deploySliceOrdered()}
+	require.Len(t, b.Bids, 1000)
```

Comment:
🚧 If this fails it will use all the runes in the internet to tell you about it. Prefer Equal.
But, do we even need this test, because it seems like this fixture is baked in.
```suggestion
	require.Equal(t, 1000, len(b.Bids), "deploySliceOrdered() should return correct number of bids")
```

Reply:
Will RM